### ISSUE TITLE
format for directories to ignore...

### ISSUE THREAD

**Issue by lavaguy1** [felipegiacomozzi/the-trove-downloader#26]
Can someone please give an example of the format for the parameters to option 2) Directories to Ignore? In particular, what must be entered to NOT download the /Books/%20Collections/ directory, or other directories beginning with a space.

Also, can wildcards somehow be used, like to say "do not download any directory beginning with the letter 'a'"?

Thanks!

**Comment by kdiii1** [comment]
You would enter Collections/ to make it ignore the Collections directory

**Comment by lavaguy1** [comment]
> You would enter Collections/ to make it ignore the Collections directory

Yeah, that was my first thought, too. But it doesn’t work. Probably because the directory name on the server begins with „%20“

**Comment by felipegiacomozzi** [comment]
Hello @lavaguy1 ,
I just released a new version that fixes this.
It needed a cleanup on the folder name of the website to work, so you don`t need to put the extra spaces at the start or at the end.
Typing Collections/ now should work to filter it out.

https://github.com/felipegiacomozzi/the-trove-downloader/releases/tag/1.0.16

**Comment by lavaguy1** [comment]
i'll give it a try. thanks!
is the trailing comma still necessary - i read somewhere that it is...

And Merry Christmas!

**Comment by lavaguy1** [comment]
i'll give it a try. thanks!
is the trailing comma still necessary - i read somewhere that it is...

And Merry Christmas!

**Comment by lavaguy1** [comment]
That seems to have donethe trick. It's skipping now, using:

Collections/,

Thanks!

**Comment by lavaguy1** [comment]
That worked great! Second question... how would you properly exclude a directory called /Dark Eye, The/ ?
